client/src/app/pages/source-platforms/usePlatformKindList.ts

```diff
@@ -1,7 +1,37 @@
+import { OptionWithValue } from "@app/components/SimpleSelect";
+import React from "react";
+
 export const DEFAULT_KIND = "cloudfoundry";
 
-const KIND_LIST = [DEFAULT_KIND]; // Add more provider kinds here, or use a list from the API
+// Extend this map as new platform kinds are added
+const KIND_MAP = {
```

Comment:
```suggestion
const KIND_MAP: Record<string, { label: string }> = {
```

Adding the type here makes the getDisplayLabel function much easier.